sql_metadb/derived_tables/instance_ext.sql

```diff
@@ -0,0 +1,122 @@
+-- Create an extended version of the instance table where the names for identifiers are included
+-- This derived table doesn't include all the properties extracted from the json objects
+-- For example, electronic access, notes, publication, series, or edition statements have their own 
+-- derived tables.
+DROP TABLE IF EXISTS instance_ext;
+
+CREATE TABLE instance_ext AS 
+WITH expanded_inst AS (
+	SELECT 
+		i.id AS instance_id,
+		jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,
+		jsonb_extract_path_text(i.jsonb, '_version') AS record_version,

```

Comment:
Do not think `_version` is required.

My understanding is records in `folio_inventory.instance` are always current; other versions are listed in separate `instance_` table.

Reply:
I wasn't sure. There are issues with the property generation. I guess if people really want version then they can query it.